traffic_portal/app/src/common/modules/dialog/_dialog.scss

```diff
@@ -45,3 +45,16 @@
 .dialog-20 .modal-dialog {
   width: 20%;
 }
+
+.fieldset-body {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-column-gap: 1.5em;
+  grid-row-gap: 5px;
+  justify-items: start;
+  align-items: center;
+  ​
```

Comment:
invalid US-ASCII byte on this line. I think Slack did it to you, they do a lot of dumb things with copy/paste. Try deleting this line and re-inserting it.

Reply:
done.